evap/static/css/_quick-review.scss

```diff
@@ -18,13 +18,19 @@
         .fas {
             cursor: pointer;
         }
+        .badge {
```

Comment:
I would suggest to extend `.slider-side` so that it will manipulate `visibility`, that way the badges will be disabled without touching `display`. If you can sacrifice the opacity transition, then things are very easy, if you want to keep them, then you [could use animations](https://stackoverflow.com/questions/30064932/hide-element-after-fade-out-using-only-css) (you don't need vendor prefixes anymore).

Reply:
what is the advantage of your suggested solution compared to setting `display` to `none`?

Reply:
The disadvantage here is that you try to fix a bug by overwriting the child, instead of changing the parent which has the show / hide behaviour. But the children are independent of this behaviour and should only be adjusted when it's really necessary.

Also, if I set the transition duration to a higher value, I see that you already ditched the fade-out and fade-in effects for the badges. Thus, I think something like this would do the same job without child selector: (locally tested)

```css
.slider-item {
    visibility: hidden;
    &.shown {
        visibility: visible;
    }
}
```

(If you apply this, please also remove the `opacity` stuff as it won't be in effect after this)